assets/css/main.scss

```diff
@@ -323,22 +323,22 @@ body.location-pasteur {
 }
 
 
-.rna-intro-header {
-    padding-top: 50px;
-    padding-bottom: 50px;
-    color: #000000;
-    background: url(/assets/media/rna_workbench_logo_bg.png) no-repeat center center;
-    background-size: cover;
+.tooltable {
+    td:nth-child(1) {
+        width: 15%;
+    }
+
+    td:nth-child(3) {
+        width: 15%;
+    }
 }
 
-.rna-intro-message {
-    position: relative;
-    padding-top: 5%;
-    padding-bottom: 30%;
-    text-align: right;
-    
-    h1 {
-        margin: 0;
-        font-size: 5em;
+.rna-intro-right {
```

Comment:
@bebatut general Q, are we fine adding this in the main.css or should we put this into the markdown?

Reply:
I would prefer to put in the `main.css` than in the markdown